### PR TITLE
GEODE-2741: Adding a warning for 64bit Windows Tools sets

### PR DESCRIPTION
**Pull request by mhansonp**
- Need to include -Thost=x64 in the configuration step

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -17,6 +17,15 @@ project(nativeclient)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/../cmake)
 
+if(CMAKE_GENERATOR MATCHES Win64*)
+  if ((CMAKE_GENERATOR MATCHES "Visual Studio") AND (CMAKE_GENERATOR_TOOLSET STREQUAL ""))
```
Comment (pivotal-jbarrett) [inline src/CMakeLists.txt]: Why not just set the `CMAKE_GENERATOR_TOOLSET`?

**Comment by pivotal-jbarrett** [comment]
@mhansonp please follow up or close this pull request.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -17,6 +17,15 @@ project(nativeclient)
...
+    message(WARNING "GEODE expects that a user must provode -Thost=x64 if you are using a"
```
Comment (vjr) [inline src/CMakeLists.txt]: Typo - "provode" should be "provide" ?

**Comment by pivotal-jbarrett** [comment]
@mhansonp you have conflicts, can you rebase and force push.

**Comment by mhansonp** [comment]
Sorry, I will fix that shortly. I thought that went to my fork only….

**Comment by pivotal-jbarrett** [comment]
@mhansonp Could you please rebase since the major re-org.

**Pull request merged**